pkg/kubelet/stats/cri_stats_provider_test.go

```diff
@@ -1072,3 +1073,28 @@ func TestGetContainerUsageNanoCores(t *testing.T) {
 		assert.Equal(t, test.expected, cached, test.desc)
 	}
 }
+
+func TestExtractIDFromCgroupPath(t *testing.T) {
+	tests := []struct {
+		cgroupPath string
+		expected   string
+	}{
+		{
+			cgroupPath: "/kubepods/burstable/pod2fc932ce-fdcc-454b-97bd-aadfdeb4c340/9be25294016e2dc0340dd605ce1f57b492039b267a6a618a7ad2a7a58a740f32",
+			expected:   "9be25294016e2dc0340dd605ce1f57b492039b267a6a618a7ad2a7a58a740f32",
+		},
+		{
+			cgroupPath: "/kubepods.slice/kubepods-burstable.slice/kubepods-burstable-pod2fc932ce_fdcc_454b_97bd_aadfdeb4c340.slice/cri-containerd-aaefb9d8feed2d453b543f6d928cede7a4dbefa6a0ae7c9b990dd234c56e93b9.scope",
```

Comment:
@haircommander does CRI-O cgroup naming also match this pattern?

Reply:
yes!

Reply:
though obviously the Base is cri-o-$id.scope

Reply:
Hi @bobbypage  @haircommander , will the suffix of systemd-style cgroupPath always be `.scope` ? If yes, I can add check like `if strings.HasSuffix(id, ".scope") `

Reply:
I think so because runc will create an underlying systemd scope (https://www.freedesktop.org/software/systemd/man/systemd.scope.html) for the container -- https://github.com/opencontainers/runc/blob/master/docs/systemd.md